LL firepower rest api106 JSON

ReneeBornstein opened the pull request:
created titles for table of contents

ReneeBornstein committed:
created titles in JSON for table of contents

ReneeBornstein committed:
added 101 to first title

ReneeBornstein committed:
created titles for table of contents

ReneeBornstein committed:
created titles for table of contents

ReneeBornstein committed:
added titles for table of contents

ReneeBornstein committed:
created titles for table of contents

ReneeBornstein committed:
created titles for table of contents

annegentle commented on labs/firepower-restapi-106/firepower-restapi-106.json:
```diff
@@ -30,9 +30,11 @@
     "files": [{
-        "title": "1.md"
+        "file": "1.md",
+        "title": "FMC - REST - API - 106"
```
Doesn't need the dashes.

annegentle commented on labs/firepower-restapi-106/firepower-restapi-106.json:
```diff
@@ -30,9 +30,11 @@
     }, {
-        "title": "2.md"
+        "file": "2.md",
+        "title": "FMC REST API Objects"
```
Make sure the title matches what's in the Markdown file - it seems like it would be objects, lowercase.

ReneeBornstein replied:
No, actually, in the titles, most are capitalized. Let's stay consistent...

ReneeBornstein committed:
Update firepower-restapi-106.json

removed dashes in titles
checked to make sure that md2 used the word Objects capitalized; it does

agentlecisco committed:
Update firepower-restapi-102.json
